test/Influencers/InfluencersTest.php

```diff
@@ -570,6 +570,62 @@ public function testLookup()
             'tags' => array($this->infTag, $this->infTag2)));
     }
 
+    /**
+     * @group read-only
+     * @group audience
+     */
+    public function testSearchAudience()
+    {
+        $inf = Traackr\Influencers::searchAudience([
+            'audience' => json_encode([
+                'network' => 'twitter',
+                'filters' => [
+                    [
+                        'code' => 'D02'
+                    ]
+                ]
+            ]),
+
+            'count' => 1
+        ]);
+
+        $this->assertCount(1, $inf['influencers']);
+        $this->assertEmpty($inf['influencers'][0]['post_hits']);
+    }
+
+    /**
```

Comment:
do we have any norms around the use of these groups?:

Reply:
The only group we used prior to this is the `read-only` group. It designates the tests Travis can safely run (does not alter data via the API, etc.). I've added the `audience` group to allow testers to run just the tests around the new endpoint. The `error-check` group is one I would like to see us start to use going forward to allow us to run just the tests that exercise error checking behavior -- missing parameters, etc.

Reply:
Sounds good. Perhaps these groups can be documented somewhere on the Readme.